grammars/ini.cson

```diff
@@ -1,5 +1,6 @@
 'fileTypes': [
   'cfg',
+  'conf',
```

Comment:
Ahh dang gotcha'. I should've looked at the previous PR, my bad. Is there anyway to add an option in Atom for the user to add file extensions do you think? Other packages have it but I am not sure how that would work.